changelog/exceptions-opt.dd

```diff
@@ -0,0 +1,26 @@
+Exception trapping can now be disabled via `--DRT-exopt=trap:0`
+
+Previously it was only possible to disable the trapping of exception by setting
+the global variable `rt_trapExceptions` to `false`.
+For example, you can now immediately open `gdc` at the uncaught exception:
```

Comment:
Typo `gdc` ?

Reply:
Oh. Yeah it should have been `gdb`. Fixed. Thanks!